index.html

```diff
@@ -1,108 +1,164 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
-        <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
-        <title>Web developer | Portfolio | Jitka Moreira</title>
-    </head>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
 
-    <body>
-         <!--Main nav bar-->
-        <div class="navigation">
-            <header class="container flex">
-                <h1 class="logo">jitka moreira</h1>
-                <nav>
-                    <ul>
-                        <li><a href="#about">About me</a></li> 
-                        <li><a href="#work">My work</a></li>
-                        <li><a href="#footer">Contact</a></li>
-                        <!--temporary link to my LinkedIn page-->
-                        <li><a href="https://www.linkedin.com/in/jitka-moreira-26838a73/">Resume</a></li>
-                    </ul>
-                </nav>
-            </header>
-        </div>
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+    <title>Web developer | Portfolio | Jitka Moreira</title>
```

Comment:
Love the use of pipes here! 
Consider removing one of the sections from your title however as a tab will typically not be wide enough to display all.

Reply:
Good point, thank you, will do!